### ISSUE TITLE
Add a JS-based Flash version detect

### ISSUE THREAD

**Issue by JamesMGreene** [zeroclipboard/zeroclipboard#138]
The Flash-based version detection needed to issue the "wrongFlash" event doesn't always work.  My suspicions is that it will never work for versions of Flash older than the version we targeted when we built it (due to capabilities differences).  For instance, we build ZeroClipboard for Flash 10.x, so I think users with Flash 9 (as in PR #136) will not get our Flash-based version detection error.

In other words, it looks like we need to add a JavaScript-based Flash version detection after all. :confused:
Luckily, it doesn't look _too_ hard: [swfobject.js#L47-75](https://github.com/swfobject/swfobject/blob/master/swfobject/src/swfobject.js#L47-75)

See the comments in PR #136 for a little more background.
